build/vega-lite-schema.json

```diff
@@ -354,12 +366,12 @@
           "type": "string"
         },
         "grid": {
-          "description": "A boolean flag indicating if grid lines should be included as part of the axis\n\n__Default value:__ `true` for [continuous scales](https://vega.github.io/vega-lite/docs/scale.html#continuous) that are not binned; otherwise, `false`.",
+          "description": "A boolean flag indicating if grid lines should be included as part of the axis.",
```

Comment:
Missing default value.   Review the schema diff by yourself first?

Reply:
Hmm, I added the right comment already. Let me see what's going wrong here.

Reply:
Fixed